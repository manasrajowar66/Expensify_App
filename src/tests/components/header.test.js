import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/header';

test('should render header correctly',()=>{
    const wrraper = shallow(<Header />);
    expect(wrraper).toMatchSnapshot();
});