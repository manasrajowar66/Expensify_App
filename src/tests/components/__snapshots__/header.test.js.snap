// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <[object Object]
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/create"
  >
    Create
  </[object Object]>
</div>
`;
